spec/models/user_spec.rb

```diff
@@ -151,6 +164,36 @@ def new_group2
     expect(user.groups).to eq [group]
   end
 
+  describe "pg_search_scope" do
```

Comment:
Cool! Liking the tests. This should probably be describe `"#search"` though as we're testing that method, not the library code.